src/rdkafka_proto.h

```diff
@@ -584,11 +584,30 @@ typedef struct rd_kafka_Uuid_s {
 } rd_kafka_Uuid_t;
 
 #define RD_KAFKA_UUID_ZERO                                                     \
-        { 0, 0, "" }
+        (rd_kafka_Uuid_t) {                                                    \
+                0, 0, ""                                                       \
+        }
 
 #define RD_KAFKA_UUID_METADATA_TOPIC_ID                                        \
-        { 0, 1, "" }
+        (rd_kafka_Uuid_t) {                                                    \
+                0, 1, ""                                                       \
+        }
+
 
+/**
+ * Initialize given UUID to zero UUID.
+ *
+ * @param uuid UUID to initialize.
+ */
+static RD_INLINE RD_UNUSED void rd_kafka_Uuid_init(rd_kafka_Uuid_t *uuid) {
+        memset(uuid, 0, sizeof(*uuid));
+}
```

Comment:
Shall we remove this function if we are not using it? Just a thought.

Reply:
I added it in case we had to do something more that just setting to zero when initializing a Uuid on the stack, but probably it won't be needed and we can just use `= RD_KAFKA_UUID_ZERO`